### PR TITLE
Copilot workflow changes

### PR DESCRIPTION
**Pull request by gondzo**

### CODE REVIEW

File: connect/connectNotificationServer.js [hunk 1]
```diff
@@ -307,6 +335,7 @@ const handler = (topic, message, logger, callback) => {
       //       - check that event has everything required or throw error
       getNotificationsForTopicStarter(eventConfig, message.topicId),
```
Comment (vikasrohit) [inline connect/connectNotificationServer.js]: @gondzo we might reuse the logic of `originator`, which I mentioned in other comment, here to be generic enough to send notifications to the originator.

File: connect/config.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ module.exports = {
   // These variables are currently being used to retrieve above role members using API V3 `/roles` endpoint.
   // As soon as this endpoint is replaced with more suitable one, these variables has to be removed if no need anymore.
   CONNECT_MANAGER_ROLE_ID: 8,
+  CONNECT_COPILOT_MANAGER_ROLE_ID: 113,
```
Comment (vikasrohit) [inline connect/config.js]: @gondzo I forgot how do we manage these ids when we test the application in dev env?

Comment (gondzo) [reply connect/config.js]: We just use what is set in the admin app - role ids should be consistent between dev and prod

File: emails/src/partials/project-team.html [hunk 3]
```diff
@@ -63,15 +63,15 @@
                         {{#if [notifications.connect.project.member.invite.requested]}}
                             You are requested to add <strong>{{userFullName}}</strong> as a copilot
                         {{/if}}
-                        {{#if [notifications.connect.project.member.copilot.added]}}
+                        {{#if [notifications.connect.project.member.invite.approved]}}
                             {{#if [creator]}}
```
Comment (RishiRajSahu) [inline emails/src/partials/project-team.html]: @gondzo I think we need to rename this creator as well, need not we ?

Comment (gondzo) [reply emails/src/partials/project-team.html]: this is done already @RishiRajSahu

**Comment by gondzo** [comment]
@RishiRajSahu merge conflicts resolved

**Pull request merged**